lib/happo/views/diffs.erb

```diff
@@ -10,52 +10,27 @@
     <style>
       <%= Happo::Utils.css_styles %>
     </style>
-  </head>
-
-  <body>
-    <header class="header">
-      <h1 class="header_title">
-        <%= Happo::Utils.page_title(diff_images, new_images) %>
-      </h1>
-      <div class="header__time">Generated: <%= Time.now %></div>
-    </header>
-
-    <main class="main">
-      <% if diff_images.count > 0%>
-        <h2 id="diffs">
-          <a class="anchored" href="#diffs">
-            Diffs (<%= diff_images.count %>)
-          </a>
-        </h2>
 
-        <% diff_images.each do |image| %>
-          <h3 id="<%= Happo::Utils.image_slug(image) %>">
-            <a class="anchored" href="#<%= Happo::Utils.image_slug(image) %>">
-              <%= Rack::Utils.escape_html(image[:description]) %> @ <%= image[:viewport] %>
-            </a>
-          </h3>
+    <script src="https://npmcdn.com/react@15.3.1/dist/react.min.js"></script>
+    <script src="https://npmcdn.com/react-dom@15.3.1/dist/react-dom.min.js"></script>
+    <script src="https://npmcdn.com/babel-core@5.8.38/browser.min.js"></script>
```

Comment:
We could have a fallback to another cdn hosted version. Falling back to a local file would mean always uploading the fallback for every serialized diffs page. 

This would work I think:
```js
window.React || document.write('<script src="fallback-url" />');
```